Mugs_index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Filling Cups with Liquids</title>
+    <style>

```

Comment:
This CSS in line should be extracted to external brand new file.

Reply:
Hello, 
It was done after our meeting, should I leave a comment each time or you already got a notification about new commit?